src/app/centers/centers.component.html

```diff
@@ -1,53 +1,60 @@
-            <i class="fa fa-circle fa-2x"> </i>
-        </div> </td>
-        
-      </ng-container>
-
-
-
-      <!-- Status Column -->
-      <ng-container matColumnDef="office">
-        <th mat-header-cell *matHeaderCellDef mat-sort-header='office'> Office </th>
-        <td mat-cell *matCellDef="let row"> {{row?.officeName}} </td>
-      </ng-container>
-
-
-      <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
-      <tr mat-row *matRowDef="let row; columns: displayedColumns;" [routerLink]="[ ]">
-         
-      </tr>
-    </table>
-
-    <mat-paginator [pageSizeOptions]="[10, 25, 100]"></mat-paginator>
-  </div>
\ No newline at end of file
+<mat-card class="container">
+  <div fxLayout="row" fxLayoutAlign="start center">
+    <div class="search-box">
+      <mat-form-field>
+        <input matInput placeholder="Search by Name" [formControl]="name">
+      </mat-form-field>
+      <mat-form-field>
+        <input matInput placeholder="Search by ExternalID" [formControl]="externalId">
+      </mat-form-field>
+      <mat-checkbox #showClosedCenters labelPosition="after" (change)="changeShowClosedCenters()">Show Closed Centers
+      </mat-checkbox>
+    </div>
+    <div class="action-button" fxLayoutGap="25px">
+      <button mat-raised-button color="primary">
+        <fa-icon icon="download"></fa-icon>&nbsp;&nbsp;Import Center
+      </button>
+      <button mat-raised-button color="primary">
+        <fa-icon icon="plus"></fa-icon>&nbsp;&nbsp;Create Center
+      </button>
+    </div>
+  </div>
+
+  <table mat-table [dataSource]="dataSource" matSort>
+
+    <ng-container matColumnDef="name">
+      <th mat-header-cell *matHeaderCellDef mat-sort-header> Name </th>
+      <td mat-cell *matCellDef="let center"> {{ center.name }} </td>
+    </ng-container>
+
+    <ng-container matColumnDef="accountNo">
+      <th mat-header-cell *matHeaderCellDef> Account # </th>
+      <td mat-cell *matCellDef="let center"> {{ center.accountNo }} </td>
+    </ng-container>
+
+    <ng-container matColumnDef="externalId">
+      <th mat-header-cell *matHeaderCellDef> ExternalID </th>
+      <td mat-cell *matCellDef="let center"> {{ center.externalId }} </td>
+    </ng-container>
+
+    <ng-container matColumnDef="status">
+      <th mat-header-cell *matHeaderCellDef> Status </th>
+      <td mat-cell *matCellDef="let row">
+        <div [ngClass]="row.active|statusLookup">
+          <fa-icon icon="stop"></fa-icon>
+        </div>
+      </td>
+    </ng-container>
+
+    <ng-container matColumnDef="officeName">
+      <th mat-header-cell *matHeaderCellDef mat-sort-header> Office Name </th>
+      <td mat-cell *matCellDef="let center"> {{ center.officeName }} </td>
+    </ng-container>
+
+    <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+    <tr mat-row *matRowDef="let row; columns: displayedColumns;" [routerLink]="[ ]"></tr>
```

Comment:
```suggestion
    <tr mat-row *matRowDef="let row; columns: displayedColumns;"></tr>
```
Remove this as its not required for the time being.